frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -0,0 +1,173 @@
+import { getSegmentBoundingBoxes, getSegmentVolumes } from "admin/admin_rest_api";
+import { Modal, Spin, Table } from "antd";
+import saveAs from "file-saver";
+import { formatNumberToUnit, formatNumberToVolume } from "libs/format_utils";
+import { useFetch } from "libs/react_helpers";
+import { Unicode, Vector3 } from "oxalis/constants";
+import { getResolutionInfo } from "oxalis/model/accessors/dataset_accessor";
+import { Segment } from "oxalis/store";
+import React from "react";
+import { TreeNode } from "./segments_view_helper";
+import { api } from "oxalis/singletons";
+
+const SEGMENT_STATISTICS_CSV_HEADER =
+  "segmendId,segmentName,groupId,groupName,volumeInVoxel,volumeInNm3,boundingBoxTopLeftPositionX,boundingBoxTopLeftPositionY,boundingBoxTopLeftPositionZ,boundingBoxSizeX,boundingBoxSizeY,boundingBoxSizeZ";
+
+const { ThinSpace } = Unicode;
+
+type Props = {
+  onCancel: (...args: Array<any>) => any;
+  isOpen: boolean;
+  tracingId: any;
+  tracingStoreUrl: any;
+  visibleSegmentationLayer: any;
+  relevantSegments: Segment[];
+  parentGroup: number;
+  groupTree: TreeNode[];
+};
+
+type SegmentInfo = {
+  segmentId: number;
+  segmentName: string;
+  groupId: number | undefined | null;
+  groupName: string;
+  volumeInNm3: number;
+  formattedSize: string;
+  volumeInVoxel: number;
+  boundingBoxTopLeft: Vector3;
+  boundingBoxPosition: Vector3;
+};
+
+const exportStatisticsToCSV = (
+  segmentInformation: Array<SegmentInfo>,
+  tracingId: string,
+  groupId: number,
+) => {
+  if (segmentInformation.length < 0) {
+    return;
+  }
+  const segmentStatisticsAsString = segmentInformation
+    .map(
+      (segmentInfo) =>
+        `${segmentInfo.segmentId},${segmentInfo.segmentName},${segmentInfo.groupId},${segmentInfo.groupName},${segmentInfo.volumeInVoxel},${segmentInfo.volumeInNm3},${segmentInfo.boundingBoxTopLeft},${segmentInfo.boundingBoxPosition}`,
```

Comment:
I'd write it like this `[segmentInfo.segmentId, segmentInfo.segmentName, ...].join(",")`

Reply:
I had to change `tsconfig`in order to use `replaceAll`(>=only with es2021), I hop that's okay!

Reply:
I think, it's fine, but two things:

- please add a usage for `replaceAll` in `frontend/javascripts/libs/browser_feature_check.tsx` so that we can warn users about it
- in tsconfig you used es2022. however, you wrote above that es2021 would also suffice. I'd go with 2021 to keep things as backwards-compatible as possible for now (of course, it doesn't make a difference for the compilation output now, but other incompatibilities might sneak in later)